lib/signs_ui/messages/sign_content.ex

```diff
@@ -52,15 +52,15 @@ defmodule SignsUi.Messages.SignContent do
   defparsec(:parse_command, command)
 
   @spec new(String.t(), String.t(), DateTime.t()) :: {:ok, t()} | {:error, any()}
-  def new(station, command, now \\ Timex.now()) when is_binary(station) do
+  def new(station, command, now \\ DateTime.now!("Etc/UTC")) when is_binary(station) do
```

Comment:
`DateTime.utc_now()` I think would be the preferred way to do this (here and in the various test files)